README.md

```diff
@@ -139,13 +144,15 @@ Serializes any JavaScript value into a JSON-compatible object.
 #### Examples
 
 ```js
+import superjson from 'superjson';
+
 const object = {
   normal: 'string',
   timestamp: new Date(),
   test: /superjson/,
 };
 
-const { json, meta } = serialize(object);
+const { json, meta } = superjson.serialize(object);
```

Comment:
Same as above: I think the superjson import really adds a lot of noise.

Reply:
Could you revert these changes too, just as the other superjson imports further below?